### DISCUSSION TITLE
Start Bluetooth Service

### DISCUSSION THREAD

**Discussion by husseinkev** [PowerShell/PowerShell#14805] [Q&A]
Hi,
Is it possible to start a Bluetooth service to all user's accounts? We have a default stopped Bluetooth service in our environment; however, by running "Start-Service -name BluetoothUserService_6dc34" only starts the service on the current user. I want to be able to run a script that will run the service of the Bluetooth without modifying the suffix every time since the suffix is based on the user's account. I'm okay with running the same script on multiple users, I just don't want to keep modifying the suffix every time, I just want to run it and it picks up directly if that's possible.

**Comment by kvprasoon** [comment]
You can use wildcard
```
Start-Service -Name "BluetoothUserService*" -verbose
```
